### PR TITLE
CFP post

### PR DESCRIPTION
**Pull request by jcjudkins**
## Description

Call for Proposals Post

Need to update dates

### CODE REVIEW

**Comment by jcjudkins** [comment]
- [x] Need to update talk submission links.
- [x] Need to add Opportunity Grant Link.

File: _posts/2024-02-27-announcing-call-for-proposals.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+date: 2024-02-12 06:00:00
+layout: post
+image: /static/img/blog/speaker_cfp.png
+post_photo_url: /static/img/blog/speaker_cfp.jpg
+post_photo_alt: "Abigail Mesrenyame Dogbe speaking at DjangoCon US 2023"
+title: "The Call for Proposals for DjangoCon US 2024 Is Now Open!"
+title: The Call for Proposals for DjangoCon US 2024 Is Now Open!
+---
+<!-- Todo: Need dates and where to submit proposals. Also need to update the link to the CFP. -->
+We are excited that DjangoCon US 2024 is now open for talk submissions: [call for proposals](https://pretalx.com/djangocon-2023/cfp)! The deadline for submissions is April 24th, 2024 at [6 PM EDT](https://time.is/0600PM_15_May_2023_in_Durham,_United_States?DjangoCon_US_CFP_Closes).
```
Comment (drewbrew) [inline _posts/2024-02-27-announcing-call-for-proposals.md]:
```suggestion
We are excited that DjangoCon US 2024 is now open for talk submissions: [call for proposals](https://pretalx.com/djangocon-us-2024/cfp)! The deadline for submissions is April 24th, 2024 at [Noon EDT](https://time.is/1200PM_24_April_2024_in_Durham,_United_States?DjangoCon_US_CFP_Closes).
```

File: _posts/2024-02-27-announcing-call-for-proposals.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+author: Jason Judkins
+category: General
+date: 2024-02-12 06:00:00
```
Comment (drewbrew) [inline _posts/2024-02-27-announcing-call-for-proposals.md]:
```suggestion
date: 2024-02-28 06:00:00
```

File: _posts/2024-02-27-announcing-call-for-proposals.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+title: The Call for Proposals for DjangoCon US 2024 Is Now Open!
+---
+<!-- Todo: Need dates and where to submit proposals. Also need to update the link to the CFP. -->
+We are excited that DjangoCon US 2024 is now open for talk submissions: [call for proposals](https://pretalx.com/djangocon-2023/cfp)! The deadline for submissions is April 24th, 2024 at [6 PM EDT](https://time.is/0600PM_15_May_2023_in_Durham,_United_States?DjangoCon_US_CFP_Closes).
+
+We invite you to submit your proposal no matter your background or experience level with Django. Proposals can be from a wide range of topics; non-Django and community topics are welcome. You can look at our [talk schedule](https://2023.djangocon.us/talks/) from last year for reference.
+
+We fancy first-timers! If you haven’t spoken at a conference or given a tutorial before, this is your invitation to do so. Don’t let the idea that you’re not famous or an expert stop you from submitting. It certainly won’t stop us from selecting your talk or tutorial and it won’t stop the audience from enjoying it! Please contact one of our amazing [speaker mentors](https://2023.djangocon.us/speaking/speaker-resources/) to help you on your way.
```
Comment (drewbrew) [inline _posts/2024-02-27-announcing-call-for-proposals.md]:
```suggestion
We fancy first-timers! If you haven’t spoken at a conference or given a tutorial before, this is your invitation to do so. Don’t let the idea that you’re not famous or an expert stop you from submitting. It certainly won’t stop us from selecting your talk or tutorial and it won’t stop the audience from enjoying it! Please contact one of our amazing [speaker mentors](https://2024.djangocon.us/speaking/speaker-resources/) to help you on your way.
```

We'll merge the CFP page before this one launches so it's fine to link to it here

File: _posts/2024-02-27-announcing-call-for-proposals.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+<!-- Todo: Add link to correct opportunity grant form. -->
+Plus there are perks! Presenters get free admission to DjangoCon US! Grants to assist with your travel and lodging expenses are available as well. Fill out the [Opportunity Grant](https://docs.google.com/forms/d/1yc1HqqcJwi669vxTTs1daJbTeiNYwqM2h3gIgqbpWVE/viewform) form by May 15th, 2023. Decision notifications will be sent by June 28, 2023.
```
Comment (drewbrew) [inline _posts/2024-02-27-announcing-call-for-proposals.md]:
```suggestion
Plus there are perks! Presenters get free admission to DjangoCon US! Grants to assist with your travel and lodging expenses are available as well. Fill out the [Opportunity Grants](https://forms.gle/Pi12J6vFQHq2CSAy5) form by April 24th, 2024. Decision notifications will be sent by June 14, 2024.
```

**Review by drewbrew — APPROVED** [review]
I'm pretty sure I have these links correct. Let's not merge until we're sure we have the pieces lined up.

**Comment by tim-schilling** [comment]
@drewbrew maybe we should rebase this on your branch?

**Comment by drewbrew** [comment]
I like that idea

**Review by tim-schilling — APPROVED** [review]

**Pull request merged**